setup.py

```diff
@@ -2,7 +2,7 @@
 from setuptools import setup
 
 setup(
-    name='rlgarage',
+    name='garage',
```

Comment:
unfortunately garage is taken on PyPI so I have named it rlgarage for now.

Reply:
but does that mean our package will be named rlgarage? That will change all our example files